test/functional/page_objects/common_page.ts

```diff
@@ -164,116 +202,84 @@ export function CommonPageProvider({ getService, getPageObjects }) {
       } else {
         appUrl = getUrl.noAuth(config.get('servers.kibana'), {
           pathname: `${basePath}/app/${appName}`,
-          hash
+          hash,
         });
       }
 
       log.debug('navigating to ' + appName + ' url: ' + appUrl);
 
-      function navigateTo(url) {
-        return retry.try(function () {
+      await retry.tryForTime(defaultTryTimeout * 2, async () => {
```

Comment:
I decreased retry timeout from **6** min to **4** min since it should be fairly enough time for navigation + login, in case of failure we should get a screenshot with stack trace instead of mocha timeout